### ISSUE TITLE
refactor all code package to com.huaweicloud

### ISSUE THREAD

**Issue by liubao68** [huaweicloud/spring-cloud-huawei#61]
Currently is 
```
org.springframework.cloud.huawei.config
```

and

```
org.springframework.cloud.dtm.consumer.feign
```

These are not very good. Change to the following

```
com.huaweicloud.config
```

and

```
com.huaweicloud.dtm
```

**Comment by tianxiaoliang** [comment]
Good suggestion

**Comment by liubao68** [comment]
fixed in 1.0.0
